server/routes/CommentsRoutes.js

```diff
@@ -4,7 +4,7 @@ import {list,show,create,update,remove} from "../controllers/CommentsController"
 
 
 router.get("/comments", list);
-router.get("/comments/:id", show);
+router.get("/comment/:id", show);
```

Comment:
don't change the naming convention pattern
the path should always be the plural of whatever it is. 
This is standard operating procedure